src/components/modal/_modal.scss

```diff
@@ -16,7 +16,7 @@
 @include exports('modal') {
   .bx--modal {
     @include reset;
-    @include font-family;
+    @include helvetica;
```

Comment:
@joshblack Did you really mean to make changes here?

Reply:
Nope! Thanks for catching 😄